app/views/portal/annotations.json.erb

```diff
@@ -0,0 +1 @@
+<%= raw record_annotations.to_json %>
```

Comment:
It would still need to run the annotations through the `record_annotations` method, so on second thoughts let's just leave this simple view as-is so as not to be calling view-related methods in the controller.